internal/kafka/test/integration/kafka_list_search_order_test.go

```diff
@@ -158,6 +176,22 @@ func Test_KafkaListSearchAndOrderBy(t *testing.T) {
 			expectedTotal: 3,
 			notContains:   []string{nonExistentKafkaName},
 		},
+		{
+			name:          "Filter cluster_id not in ('')",
+			searchOpts:    &public.GetKafkasOpts{Search: optional.NewString("cluster_id not in ('')")},
+			wantErr:       false,
+			expectedSize:  3,
+			expectedTotal: 3,
+			notContains:   []string{nonExistentKafkaName, mockKafkaName4},
+		},
```

Comment:
@ziccardi thanks for adding this.

Is it possible to add the inverse test case? e.g `cluster_id IN (cluster_id_1, cluster_id_2)` and ensure only Kafka within the organization are returned? Thanks

Reply:
done!